libkbfs/kbfs_ops.go

```diff
@@ -256,26 +256,53 @@ func (fs *KBFSOpsStandard) getOpsByHandle(ctx context.Context,
 	return ops
 }
 
+func (fs *KBFSOpsStandard) getMDByHandle(ctx context.Context,
+	tlfHandle *TlfHandle) (rmd ImmutableRootMetadata, id tlf.ID, err error) {
+	fbo := func() *folderBranchOps {
+		fs.opsLock.Lock()
+		defer fs.opsLock.Unlock()
+		return fs.opsByFav[tlfHandle.ToFavorite()]
+	}()
+	if fbo != nil {
+		lState := makeFBOLockState()
+		rmd = fbo.getHead(lState)
+	}
+	if rmd == (ImmutableRootMetadata{}) {
+		_, rmd, id, err = fs.getOrInitializeNewMDMaster(
+			ctx, fs.config.MDOps(), tlfHandle, true)
+		if err != nil {
+			return ImmutableRootMetadata{}, tlf.ID{}, err
+		}
+		if fbo != nil {
```

Comment:
To get any benefit, I think if `fbo == nil && !initialized`, you're going to have to make a new `fbo` here (e.g.:
```go
fb := FolderBranch{Tlf: id, Branch: MasterBranch}
fbo := fs.getOpsByHandle(ctx, tlfHandle, fb)
```

Otherwise you'll still be fetching from the server each time, unless you are creating the TLF for the first time.